### ISSUE TITLE
Code coverage

### ISSUE THREAD

**Issue by leto** [fossasia/ots15-companion#3]
Similar to our current Travis CI badge, it would be helpful to know our "code coverage".

I suggest Coveralls which is free for Free+Open source projects:

https://coveralls.io

This issue is to add a code coverage badge to our README.

Tasks
- [ ] Integrate github repo to Coveralls (requires certain Github permissions)
- [ ] Make sure Coveralls can understand our test suite and run it correctly
- [ ] Potentially make a new commit and push it to get our first code coverage run
- [ ] Verify that our code coverage badge is working on our README

If you don't understand exactly what each task above means, please clarify before starting your work.

I expect this to take about 1 hour of focused work for a GSoC student, so if it is taking longer than that, please ask for help on this issue. /cc @mariobehling

**Comment by championswimmer** [comment]
@Duke, we are not at this point using automated test suites, so I am not
sure how exactly code coverage would work out ?

# 

  Arnav Gupta (championswimmer)
  LinkedIn http://in.linkedin.com/in/arnavgupta/ | Web
http://championswimmer.in | Blog http://blog.championswimmer.in |
Twitter http://twitter.com/championswimmer | Facebook
http://facebook.com/championswimmer | Github
http://github.com/championswimmer

**Comment by leto** [comment]
Yes, if there are no automated tests at all, then hold off on this until they exist.

**Comment by leto** [comment]
That means we need automated tests ASAP :smiling_imp:

**Comment by creativepsyco** [comment]
Is this for testing in the Android app? Then perhaps I can help, I have some luck running automated tests on Jenkins and CircleCI.

**Comment by leto** [comment]
@creativepsyco yes, this is for running any kind of tests that can help us see if our code quality is going up or down. Your help is much appreciated!
